src/Fantomas/Utils.fs

```diff
@@ -152,6 +152,18 @@ module List =
         | [ _ ] -> false
         | _ -> true
 
+    let partitionWhile (f: int -> 'a -> bool) (xs: 'a list) : ('a list * 'a list) =
+        let rec go i before after =
+            match after with
+            | head :: tail ->
+                match f i head with
+                | true -> go (i + 1) (head :: before) tail
+                | false -> List.rev before, after
+            | [] -> List.rev before, after
+
+        go 0 [] xs
+
+
```

Comment:
Super nitpick, but could you remove one of the blank lines here 😅?

Reply:
Haha, of course! I remember reading something about Fantomas not fixing the number of new lines between syntax elements...? Can't remember why though. Anyway, fixed.

Reply:
No, it will currently keep the newlines you had in the source code.